OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -41,7 +41,7 @@
     public DiscussionPostsAdapter(Context context, IEdxEnvironment environment) {
         super(context, R.layout.row_discussion_thread, environment);
         edx_brand_primary_base = context.getResources().getColor(R.color.edx_brand_primary_base);
-        edx_grayscale_neutral_dark = context.getResources().getColor(R.color.edx_brand_gray_base);
+        grey_medium = context.getResources().getColor(R.color.grey_medium);
```

Comment:
Why we are changing it to `grey_medium`?

Reply:
instead of using this, we can use `edx_brand_gray_accent` with value `#707070`

Reply:
replaced.